ReactNativeClient/lib/reducer.js

```diff
@@ -557,6 +558,12 @@ const reducer = (state = defaultState, action) => {
 				newState.decryptionWorker = decryptionWorker;
 				break;
 
+			case 'LOAD_CUSTOM_CSS':
+
+				newState = Object.assign({}, state);
+				newState.customCss = '<style>' + action.css + '</style>\n';
```

Comment:
Let's put only CSS in this variable without any <style> tags so that it can later be combined with other CSS (or minified etc.) if needed. So it would be `newState.customCss =action.css;`

and only add the style tags at the last moment:

`bodyToRender = '<style>' + this.props.customCss + '</style>' + bodyToRender;`